internal/report.go

```diff
@@ -501,3 +502,26 @@ func writeHeading(w *bufio.Writer, s string) {
 		s, "\n",
 		"----------------------------\n"}, ""))
 }
+
+func PrintSourceType(ty schema.Type) string {
+	s := ty.Name
```

Comment:
@nevinheintze Should I move it to schema/schema.go ? or This is good?

Reply:
Yes, 100% agreed, it should be moved to schema/schema.go.

I actually made the same comment when reviewing a few hours ago, but I don't see the comment! Must have messed up somehow.

In short, please go ahead and make that change -- the code logically belongs in schema.go.

Reply:
oh ok!
I have moved this function to schema/schema.go. Please review and merge this PR if everything else looks good.